### ISSUE TITLE
accelerate3d breaks atlas builds

### ISSUE THREAD

**Issue by stefco** [stefco/geco_vm#62]
```
Packer v0.10.0

virtualbox-iso output will be in this color.

==> virtualbox-iso: Downloading or copying Guest additions
    virtualbox-iso: Downloading or copying: file:///usr/share/virtualbox/VBoxGuestAdditions.iso
==> virtualbox-iso: Downloading or copying ISO
    virtualbox-iso: Downloading or copying: http://releases.ubuntu.com/12.04.5/ubuntu-12.04.5-server-amd64.iso
    virtualbox-iso: Download progress: 1%
    virtualbox-iso: Download progress: 14%
    virtualbox-iso: Download progress: 29%
    virtualbox-iso: Download progress: 42%
    virtualbox-iso: Download progress: 56%
    virtualbox-iso: Download progress: 71%
    virtualbox-iso: Download progress: 83%
    virtualbox-iso: Download progress: 97%
==> virtualbox-iso: Starting HTTP server on port 8219
==> virtualbox-iso: Creating virtual machine...
==> virtualbox-iso: Creating hard drive...
==> virtualbox-iso: Creating forwarded port mapping for communicator (SSH, WinRM, etc) (host port 3276)
==> virtualbox-iso: Executing custom VBoxManage commands...
    virtualbox-iso: Executing: modifyvm geco-vm-gui --vram 256
    virtualbox-iso: Executing: modifyvm geco-vm-gui --memory 2048
    virtualbox-iso: Executing: modifyvm geco-vm-gui --accelerate2dvideo on
    virtualbox-iso: Executing: modifyvm geco-vm-gui --accelerate3d on
    virtualbox-iso: Executing: modifyvm geco-vm-gui --clipboard bidirectional
    virtualbox-iso: Executing: modifyvm geco-vm-gui --draganddrop hosttoguest
    virtualbox-iso: Executing: modifyvm geco-vm-gui --cpus 2
==> virtualbox-iso: Starting the virtual machine...
    virtualbox-iso: The VM will be run headless, without a GUI. If you want to
    virtualbox-iso: view the screen of the VM, connect via VRDP without a password to
    virtualbox-iso: 127.0.0.1:5942
==> virtualbox-iso: Error starting VM: VBoxManage error: VBoxManage: error: This VM was configured to use 3D acceleration. However, the 3D support of the host is not working properly and the VM cannot be started. To fix this problem, either fix the host 3D support (update the host graphics driver?) or disable 3D acceleration in the VM settings (VERR_NOT_AVAILABLE)
==> virtualbox-iso: VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole
==> virtualbox-iso: Unregistering and deleting virtual machine...
==> virtualbox-iso: Deleting output directory...
Build 'virtualbox-iso' errored: Error starting VM: VBoxManage error: VBoxManage: error: This VM was configured to use 3D acceleration. However, the 3D support of the host is not working properly and the VM cannot be started. To fix this problem, either fix the host 3D support (update the host graphics driver?) or disable 3D acceleration in the VM settings (VERR_NOT_AVAILABLE)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole

==> Some builds didn't complete successfully and had errors:
--> virtualbox-iso: Error starting VM: VBoxManage error: VBoxManage: error: This VM was configured to use 3D acceleration. However, the 3D support of the host is not working properly and the VM cannot be started. To fix this problem, either fix the host 3D support (update the host graphics driver?) or disable 3D acceleration in the VM settings (VERR_NOT_AVAILABLE)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole

==> Builds finished but no artifacts were created.
```

**Comment by stefco** [comment]
easy fix: just don't use those vboxmanage options. they don't help anyway and those options are saved in the `Vagrantfile` anyway.

**Comment by stefco** [comment]
fixed in 1c89bb7
